tests/packages/geckodriver.bats

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bats
+
+setup() {
+  export GECKODRIVER_VERSION="0.16.1"
```

Comment:
Yeah, I wanted to get rid of that, once we have all tests converted to bats though, and have each test take care of whether it wants to use the cache or not. 

I started to add that to all other bats files, e.g. https://github.com/codeship/scripts/blob/master/tests/cache/bower.bats#L4